tests/bootstrap.php

```diff
@@ -82,11 +85,15 @@ public function install_wc() {
 
 		WC_Install::install();
 
-		// Reload capabilities after install, see https://core.trac.wordpress.org/ticket/28374
+		// Initialize the WC API extensions.
+		\Automattic\WooCommerce\Admin\Install::create_tables();
+		\Automattic\WooCommerce\Admin\Install::create_events();
```

Comment:
Why do we need to initialize the events? Maybe just create the tables is enough for unit test purposes?